### DISCUSSION TITLE
Feature Request: Structured Managed Metadata in Filter WebPart

### DISCUSSION THREAD

**Discussion by myatix** [microsoft-search/pnp-modern-search#1474] [Ideas]
Hi Guys,

It's been a while since I had the chance to look at the latest updates from v4 and it's looking really great!

I just thought I would add a Feature Request with regards to the Refiner WebPart. 

It would be really cool if you could see the same term store managed metadata structure in the Refiner WebPart as set in the Term Store.

Right now if you map a Managed Propert such as RefinableString00 to a Managed Metadata ows_SiteColumnName the refiner displays all there terms in a single level. It would be REALLY nice if the filter WebPart showed the Managed Metadata with the same structure as in the term store.

EG:
-Parent
-- Child 1
         ---Child1.1
--Child 2
        ---Child 2.1

Often when you create Managed Metadata in the term store with a particular structure it's because the data is related. If you can't see these relations/structure in the Refiner WebPart then it could lead to a very confusing user experience.

EG:
Sports Goods
--Football
----Balls
----Clothing
--Cricket
----Balls
----Clothing
--Tennis
----Balls
----Clothing
 
As the simple example shows above the structure can provide valuable information. Unfortunately this isn't currently the case if you use a the Refiner WebPart.

Then you would end up with this:

Balls
Balls
Balls
Clothes
Clothes
Clothes
Cricket
Football
Tennis

It would be really cool if when you configured a refinable managed property mapping mapped to ows_taxId_SiteColumnName then the Filter WebPart showed a structured refiner.

I hope this makes sense, because I am sure it would be a very popular feature.

Keep up the great work! :)

**Comment by FranckyC** [comment]
Hi @myatix, this has been already discussed many times. See https://github.com/microsoft-search/pnp-modern-search/discussions/1366
